sass/_aesop.scss

```diff
@@ -1,375 +1,392 @@
 }
 
 @media screen and (max-width: $width-tablet) {
-	.aesop-image-component .aesop-image-component-image.aesop-image-component-caption-left,
-	.aesop-image-component .aesop-image-component-image.aesop-image-component-caption-right {
-		text-align: center;
-	}
+  .aesop-image-component .aesop-image-component-image.aesop-image-component-caption-left,
+  .aesop-image-component .aesop-image-component-image.aesop-image-component-caption-right {
+    text-align: center;
+  }
+}
+
+.aesop-article-chapter:after,
+.aesop-content-img:after {
+  bottom: 0;
+  content: '';
+  left: 0;
+  opacity: .4;
+  position: absolute;
+  right: 0;
+  top: 0;
 }
 
 .aesop-article-chapter:after {
-    background-color: $color-dark-grey;
-    bottom: 0;
-    content: '';
-    left: 0;
-    opacity: .4;
-    position: absolute;
-    right: 0;
-    top: 0;
+  background-color: $color-dark-grey; // no background color defined by aesop component
+}
+
+.aesop-content-img:after {
+  background-color: inherit; // inherit color defined by aesop component
+}
+
+.aesop-content-component .aesop-content-comp-wrap.aesop-content-img:after {
+  display: block; // overrides display: table, and allows :after overlay to work
+}
+
+.aesop-content-img .aesop-content {
+  position: relative; // makes sure content appears above :after overlay
+  z-index: 1;
 }
 
 .aesop-cover-title {
-    @include rem(font-size, 42px);
-    @include site-width;
-    color: #fff;
-    margin-bottom: 0;
-    margin-top: 0;
-    padding: 250px 0 20px;
-    position: relative;
-    z-index: 1;
+  @include rem(font-size, 42px);
+  @include site-width;
+  color: #fff;
+  margin-bottom: 0;
+  margin-top: 0;
+  padding: 250px 0 20px;
+  position: relative;
+  z-index: 1;
 }
 
 .aesop-video-container {
-    margin: 40px auto;
-	max-width: 100%;
+  margin: 40px auto;
+  max-width: 100%;
 }
 
 .aesop-lightbox {
-	border: 0;
-	
-	img {
-		transition: opacity .2s;
-	}
-	
-    &:hover img {
-		opacity: .9;
-	}
+  border: 0;
+
+  img {
+    transition: opacity .2s;
+  }
+
+  &:hover img {
+    opacity: .9;
+  }
 }
 
 .aesop-timeline {
-	background-color: rgba(255, 255, 255, .9);
-	bottom: 0;
-	box-shadow: 0 0 1px rgba(0, 0, 0, .15);
-	left: 0;
-	position: fixed;
-    right: 0;
-	z-index: $z-index-aesop-timeline;
-	
-    ol {
-		list-style: none;
-		margin-bottom: 0;
-		text-align: center;
-		@include site-width($width-wide);
-        &:after {
-			display: none;
-		}
-	}
-	
-    li {
-        display: inline-block;
-        margin: 0 10px;
-        transition: all 200ms;
-        
-        a:hover {
-            border-color: $link-border-color;
-		}
-		
-        &.active a {
-            border-bottom-width: 2px;
-            font-weight: bold;
-        }
-	}
-    
-    &-stop {
-        padding-top: 140px;
+  background-color: rgba(255, 255, 255, .9);
+  bottom: 0;
+  box-shadow: 0 0 1px rgba(0, 0, 0, .15);
+  left: 0;
+  position: fixed;
+  right: 0;
+  z-index: $z-index-aesop-timeline;
+
+  ol {
+    list-style: none;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets